openvdb_cmd/vdb_tool/src/unittest.cpp

```diff
@@ -0,0 +1,966 @@
+// Copyright Contributors to the OpenVDB Project
+// SPDX-License-Identifier: MPL-2.0
+
+#define _USE_MATH_DEFINES
+
+#include <stdio.h>// for std::remove
+#include <string>
+#include <fstream>
+#include <set>
+
+#if defined(_WIN32)
+#include <direct.h>// for mkdir
+int mkdir_wrapper(const char *dirname) { return _mkdir(dirname); }
+#else
+#include <sys/stat.h>// for mkdir
+int mkdir_wrapper(const char *dirname) { return mkdir(dirname, 0777); }
+#endif
+
+#include "Tool.h"
```

Comment:
Local header should be included first according to OpenVDB coding standard. Please also consider adding`Parser.h`, `Parser.h`, and `Util.h`. Indeed they are included by `Tool.h` and will likely always be included there. It can be argued, however, that including header explicitly without depending on another file to include them is a good practice.